Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/models.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/admin.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/admin.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/admin.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/myproject/urls.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/myproject/urls.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Internal Server Error: /service/
Traceback (most recent call last):
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/backends/django.py", line 107, in render
    return self.template.render(context)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 969, in render_annotated
    return self.render(context)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/loader_tags.py", line 199, in render
    template = context.template.engine.select_template(template_name)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/engine.py", line 214, in select_template
    raise TemplateDoesNotExist(", ".join(not_found))
django.template.exceptions.TemplateDoesNotExist: app/navbar.html

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/contrib/auth/decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py", line 85, in service
    return render(request, 'app/service.html', {'user_info_list': user_info_list})
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/backends/django.py", line 109, in render
    reraise(exc, self.backend)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/backends/django.py", line 130, in reraise
    raise new from exc
django.template.exceptions.TemplateDoesNotExist: app/navbar.html
Internal Server Error: /service/
Traceback (most recent call last):
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/contrib/auth/decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py", line 85, in service
    return render(request, 'app/service.html', {'user_info_list': user_info_list})
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/backends/django.py", line 107, in render
    return self.template.render(context)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 171, in render
    return self._render(context)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 163, in _render
    return self.nodelist.render(context)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 1008, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 1008, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 969, in render_annotated
    return self.render(context)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/defaulttags.py", line 480, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/urls/base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/urls/resolvers.py", line 831, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'add_member' not found. 'add_member' is not a valid view function or pattern name.
/mnt/c/Coding/Web_coding/NMCNPM_project/myproject/urls.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/myproject/urls.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/myproject/urls.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/myproject/urls.py changed, reloading.
Watching for file changes with StatReloader
Not Found: /add_member/app/service.html
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /add_member/
Traceback (most recent call last):
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/contrib/auth/decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py", line 107, in add_member
    return render(request, 'app/add_member.html')  # Ensure this template exists
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: app/add_member.html
Watching for file changes with StatReloader
Internal Server Error: /add_member/
Traceback (most recent call last):
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 510, in parse
    compile_func = self.tags[command]
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/contrib/auth/decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py", line 107, in add_member
    return render(request, 'app/add_member.html')  # Ensure this template exists
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 573, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 7: 'static'. Did you forget to register or load this tag?
Internal Server Error: /add_member/
Traceback (most recent call last):
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 510, in parse
    compile_func = self.tags[command]
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/contrib/auth/decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py", line 107, in add_member
    return render(request, 'app/add_member.html')  # Ensure this template exists
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 573, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 7: 'static'. Did you forget to register or load this tag?
Internal Server Error: /add_member/
Traceback (most recent call last):
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/contrib/auth/decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py", line 96, in add_member
    FamilyMember.objects.create(
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/db/models/query.py", line 677, in create
    obj = self.model(**kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/db/models/base.py", line 543, in __init__
    _setattr(self, field.name, rel_obj)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/db/models/fields/related_descriptors.py", line 287, in __set__
    raise ValueError(
ValueError: Cannot assign "'24'": "FamilyMember.room_id" must be a "RoomUser" instance.
Internal Server Error: /add_member/
Traceback (most recent call last):
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/contrib/auth/decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py", line 96, in add_member
    FamilyMember.objects.create(
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/db/models/query.py", line 677, in create
    obj = self.model(**kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/db/models/base.py", line 543, in __init__
    _setattr(self, field.name, rel_obj)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/db/models/fields/related_descriptors.py", line 287, in __set__
    raise ValueError(
ValueError: Cannot assign "'24'": "FamilyMember.room_id" must be a "RoomUser" instance.
Internal Server Error: /add_member/
Traceback (most recent call last):
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/contrib/auth/decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py", line 96, in add_member
    FamilyMember.objects.create(
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/db/models/query.py", line 677, in create
    obj = self.model(**kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/db/models/base.py", line 543, in __init__
    _setattr(self, field.name, rel_obj)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/db/models/fields/related_descriptors.py", line 287, in __set__
    raise ValueError(
ValueError: Cannot assign "'24'": "FamilyMember.room_id" must be a "RoomUser" instance.
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /add_member/
Traceback (most recent call last):
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/contrib/auth/decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py", line 91, in add_member
    room_user = get_object_or_404(RoomUser, username=request.user.username)
NameError: name 'get_object_or_404' is not defined
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/myproject/urls.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
Using selector: EpollSelector
Using selector: EpollSelector
Using selector: EpollSelector
Using selector: EpollSelector
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
Internal Server Error: /list_member/
Traceback (most recent call last):
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 510, in parse
    compile_func = self.tags[command]
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/contrib/auth/decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py", line 60, in list_member
    return render(request, 'app/member.html', {'family_members': family_members})
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/backends/django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/junchannn/.local/lib/python3.10/site-packages/django/template/base.py", line 573, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 6: 'static'. Did you forget to register or load this tag?
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/views.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/admin.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/admin.py changed, reloading.
Watching for file changes with StatReloader
/mnt/c/Coding/Web_coding/NMCNPM_project/nmcnpmapp/admin.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /favicon.ico
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /favicon.ico
Watching for file changes with StatReloader
Not Found: /nmcnpmapp/static/css/home.css
Not Found: /nmcnpmapp/static/css/home.css
Not Found: /nmcnpmapp/static/css/home.css
Not Found: /nmcnpmapp/static/css/home.css
Not Found: /nmcnpmapp/static/css/home.css
Not Found: /nmcnpmapp/static/css/home.css
Not Found: /nmcnpmapp/static/css/home.css
Not Found: /nmcnpmapp/static/css/home.css
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.png
Not Found: /nmcnpmapp/static/img/building.png
Not Found: /nmcnpmapp/static/img/building.png
Not Found: /nmcnpmapp/static/img/building.png
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /favicon.ico
Watching for file changes with StatReloader
Not Found: /favicon.ico
Forbidden (CSRF token from POST incorrect.): /signup/
Watching for file changes with StatReloader
Internal Server Error: /login/
Traceback (most recent call last):
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\base.py", line 510, in parse
    compile_func = self.tags[command]
                   ~~~~~~~~~^^^^^^^^^
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\nmcnpmapp\views.py", line 21, in login_view
    return render(request, 'app/wait.html')
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\backends\django.py", line 79, in get_template
    return Template(self.engine.get_template(template_name), self)
                    ~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\engine.py", line 177, in get_template
    template, origin = self.find_template(template_name)
                       ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\engine.py", line 159, in find_template
    template = loader.get_template(name, skip=skip)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
        contents,
    ...<2 lines>...
        self.engine,
    )
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
                    ~~~~~~~~~~~~~~~~~~~~~^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\base.py", line 196, in compile_nodelist
    nodelist = parser.parse()
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\base.py", line 512, in parse
    self.invalid_block_tag(token, command, parse_until)
    ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\base.py", line 573, in invalid_block_tag
    raise self.error(
    ...<3 lines>...
    )
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 37: 'static'. Did you forget to register or load this tag?
Not Found: /nmcnpmapp/static/img/building.jpg
Watching for file changes with StatReloader
Not Found: /favicon.ico
Not Found: /nmcnpmapp/static/img/building.jpg
Internal Server Error: /changepassword/
Traceback (most recent call last):
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\nmcnpmapp\views.py", line 71, in changepassword
    return render(request, 'app/changepassword.html', context)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: app/changepassword.html
Internal Server Error: /changepassword/
Traceback (most recent call last):
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\contrib\auth\decorators.py", line 60, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\nmcnpmapp\views.py", line 71, in changepassword
    return render(request, 'app/changepassword.html', context)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\.venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: app/changepassword.html
Not Found: /nmcnpmapp/static/img/building.jpg
Watching for file changes with StatReloader
E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\myproject\settings.py changed, reloading.
Watching for file changes with StatReloader
E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\myproject\settings.py changed, reloading.
Watching for file changes with StatReloader
E:\TaiLieuBachKhoa\CNPM\NMCNPM_project\myproject\settings.py changed, reloading.
Watching for file changes with StatReloader
Not Found: /nmcnpmapp/static/img/building.jpg
Watching for file changes with StatReloader
Not Found: /nmcnpmapp/static/img/building.jpg
Watching for file changes with StatReloader
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Watching for file changes with StatReloader
Not Found: /nmcnpmapp/static/img/building.jpg
Watching for file changes with StatReloader
Not Found: /nmcnpmapp/static/img/building.jpg
Watching for file changes with StatReloader
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Not Found: /nmcnpmapp/static/img/building.jpg
Watching for file changes with StatReloader
